### ISSUE TITLE
Idea: let the API also send back the human readable name of property's state

### ISSUE THREAD

**Issue by flatsiedatsie** [mozilla-iot/gateway#2058]
I'm working on a voice control add-on.

When I ask the virtual leak sensor what it's state is, the api will answer only with '0'.

To turn this into a human readable response, I have to create a look-up table. _if it's a leak sensor and the response is 0, say 'dry'_, etc

Wouldn't it be an idea to also return a human readable value through the api?
- It would be less repeated code created by add-on builders.
- It would allow add-ons to still respond with the correct state even as new capabilities are introduced in the Gateway, yet before the add-on is updated to reflect those.

**Comment by mrstegeman** [comment]
There are no defined labels for property states in either the spec or the schemas. These labels will depend entirely on the language, the interface, etc. Our UI defines labels for certain states, but those label will not necessarily apply to other UIs or other languages.

**Comment by flatsiedatsie** [comment]
Alright, perhaps I can rephrase it: is there a way to find out what labels the user expects to be spoken via the API?

**Comment by mrstegeman** [comment]
No, there is not. I'm going to move this to a different repo so we can consider it for inclusion in the spec, but my initial thoughts are that this is a UI concern, so not totally relevant to the spec. However, given that we do include labels for other things, it may make sense to include.
